index.html

```diff
@@ -1,523 +1,1175 @@
+          </button>
+        </article>
+        <article class="p-0 customArticleReduced m-auto">
+          <img
+            class="w-100 mx-auto"
+            src="https://d.newsweek.com/en/full/1606968/best-nursing-homes-2020.png?w=500&h=333&f=c4c9af7aa9b2229347ab028f6dbddafa"
+            alt="aldfgdfgt"
+          />
+          <h2
+            class="p-3 pb-0 m-0 border border-secondary border-bottom-0 customText1"
+          >
+            Best Nursing Homes 2021
+          </h2>
+          <p
+            class="pt-0 p-3 border border-secondary border-top-0 customArticleIntro"
+          >
+            Tempor nostrud laborum ipsum pariatur culpa officia deserunt. Ea sit
+            fugiat fugiat non Lorem culpa. Laboris sint commodo est adipisicing
+            cillum incididunt. Exercitation nulla officia id culpa velit enim
+            quis incididunt id exercitation ea anim aute elit.
+          </p>
+        </article>
+        <article class="p-0 customArticleReduced m-auto">
+          <img
+            class="w-100 mx-auto"
+            src="https://d.newsweek.com/en/full/1587568/infection-prevention-products-banner.jpg?w=500&h=333&f=c96d696a017681c45d2f464a2ae0e23d"
+            alt="aldfgdfgdt"
+          />
+          <div class="float-left text-decoration-none customCategoryTitleDiv">
+            <a class="customCategoryTitleA" href=" ">RANKINGS</a>
+          </div>
+          <h2
+            class="p-3 pb-0 m-0 border border-secondary border-bottom-0 customText1"
+          >
+            Best Prevent Infection Produts
+          </h2>
+          <p
+            class="pt-0 p-3 border border-secondary border-top-0 customArticleIntro"
+          >
+            Eiusmod enim aute id labore ad duis anim excepteur do dolor in. Sunt
+            et enim cupidatat consequat. Qui nisi esse ullamco aliquip est
+            voluptate cillum. Aliquip consequat commodo officia officia aliquip
+            dolor ipsum minim labore exercitation minim aute enim. Fugiat
+            eiusmod dolor irure reprehenderit nostrud occaecat. Sint dolor quis
+            eu nulla aliqua et est deserunt adipisicing. Fugiat eiusmod amet
+            pariatur velit excepteur quis ut.
+          </p>
+        </article>
+      </section>
+      <div
+        class="d-flex justify-content-center align-items-center p-0 mb-3 w-100 order-4"
+      >
+        <img
+          class="w-25 m-0 mr-2 float-left customHeader customLogoLittle"
+          src="https://g.newsweek.com/www/images/logo-n1.svg"
+          alt="sdfsdf"
+        />
+        <h2 class="m-0 d-inline customText1">IN THE MAGAZINE</h2>
+      </div>
+      <section class="w-100 order-5 p-0 d-flex flex-wrap">
+        <article class="col-md-6 col-lg-3 mx-0 p-1">
+          <img
+            class="w-100"
+            src="https://d.newsweek.com/en/full/1610195/cover-july-24-2020.jpg?w=360&h=480&q=90&f=081678b318d01f44507b435d6f51963c"
+            alt="alt"
+          />
 
```

Comment:
There might be some mistake here, because this is really fixed, and I'm re-checking right now (without new changes) and is working fine!

![image](https://user-images.githubusercontent.com/63410412/88557766-05425780-d001-11ea-91c0-e6969ead6126.png)